server/routes/auth/auth.js

```diff
@@ -32,17 +34,28 @@ router.get('/auth/callback',
     // Successful authentication
     const user = await getUserDetails(req.user, req.user); // On whiten we'll take the display take from ping
 
-    /* todo detrmine isRamad acording to WITH_RAMAD_ACCESS from '.env'
-    giving priority to the .env variable, need to convert string to boolean to*/
+    const isRamad = checkIsRamad(WITH_RAMAD_ACCESS, user.is_ramad);
     
     res.cookie('user', JSON.stringify({
         upn: user.upn,
         name: user.name,
-        isRamad: user.is_ramad,
+        isRamad: isRamad,
```

Comment:
`isRamad` , as keyAndValue: keyAndValue, to keyAndValue 😃

Reply:
Done